iree/compiler/Codegen/LLVMGPU/LLVMGPULowerExecutableTarget.cpp

```diff
@@ -57,10 +57,24 @@ class LLVMGPULowerExecutableTargetPass
 };
 }  // namespace
 
+/// Verify that valid configuration is set for all ops within the compiled
+/// module.
+template <typename F>
+static LogicalResult verifyLoweringConfiguration(
```

Comment:
I know you used this from the LLVMCPU side. If you have a better solution, please use that. This was just me being lazy there. If you have a better solution, I'll use the same on the other side.

Reply:
This seems fine by me :D